[2022-06-28 14:18:01,122] {taskinstance.py:1043} INFO - Dependencies all met for <TaskInstance: first_airflow_dag.get_datetime scheduled__2022-06-28T14:17:00+00:00 [queued]>
[2022-06-28 14:18:01,129] {taskinstance.py:1043} INFO - Dependencies all met for <TaskInstance: first_airflow_dag.get_datetime scheduled__2022-06-28T14:17:00+00:00 [queued]>
[2022-06-28 14:18:01,129] {taskinstance.py:1249} INFO - 
--------------------------------------------------------------------------------
[2022-06-28 14:18:01,129] {taskinstance.py:1250} INFO - Starting attempt 1 of 1
[2022-06-28 14:18:01,129] {taskinstance.py:1251} INFO - 
--------------------------------------------------------------------------------
[2022-06-28 14:18:01,137] {taskinstance.py:1270} INFO - Executing <Task(BashOperator): get_datetime> on 2022-06-28 14:17:00+00:00
[2022-06-28 14:18:01,142] {standard_task_runner.py:52} INFO - Started process 155 to run task
[2022-06-28 14:18:01,144] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'first_***_dag', 'get_datetime', 'scheduled__2022-06-28T14:17:00+00:00', '--job-id', '219', '--raw', '--subdir', 'DAGS_FOLDER/firstdemo.py', '--cfg-path', '/tmp/tmp49w_cx1o', '--error-file', '/tmp/tmpualsm1nr']
[2022-06-28 14:18:01,145] {standard_task_runner.py:80} INFO - Job 219: Subtask get_datetime
[2022-06-28 14:18:01,181] {logging_mixin.py:109} INFO - Running <TaskInstance: first_***_dag.get_datetime scheduled__2022-06-28T14:17:00+00:00 [running]> on host ca68cb5b3870
[2022-06-28 14:18:01,216] {taskinstance.py:1448} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=first_***_dag
AIRFLOW_CTX_TASK_ID=get_datetime
AIRFLOW_CTX_EXECUTION_DATE=2022-06-28T14:17:00+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2022-06-28T14:17:00+00:00
[2022-06-28 14:18:01,217] {subprocess.py:62} INFO - Tmp dir root location: 
 /tmp
[2022-06-28 14:18:01,217] {subprocess.py:74} INFO - Running command: ['bash', '-c', 'date']
[2022-06-28 14:18:01,228] {subprocess.py:85} INFO - Output:
[2022-06-28 14:18:01,231] {subprocess.py:89} INFO - Tue Jun 28 14:18:01 UTC 2022
[2022-06-28 14:18:01,231] {subprocess.py:93} INFO - Command exited with return code 0
[2022-06-28 14:18:01,247] {taskinstance.py:1288} INFO - Marking task as SUCCESS. dag_id=first_***_dag, task_id=get_datetime, execution_date=20220628T141700, start_date=20220628T141801, end_date=20220628T141801
[2022-06-28 14:18:01,276] {local_task_job.py:154} INFO - Task exited with return code 0
[2022-06-28 14:18:01,296] {local_task_job.py:264} INFO - 0 downstream tasks scheduled from follow-on schedule check
